pkg/Cpanel/Security/Advisor/Assessors/Symlinks.pm

```diff
@@ -57,9 +58,9 @@ sub generate_advice {
 
 sub has_cpanel_hardened_kernel {
     my $self                  = shift;
-    my $hardened_kernel_state = $self->_check_for_symlink_kernel_patch();
+    my $hardened_kernel_state = ( Cpanel::Sys::Uname::get_uname_cached() )[2];
     my $ret;
-    if ( $hardened_kernel_state eq q{Symlinks_protection_enabled_for_centos6} ) {
+    if ( $hardened_kernel_state =~ m/(?:cpanel|cp)6.x86_64/ ) {
```

Comment:
Unless I'm mistaken, the `.` should be escaped.  Moreover, I recommend adding anchors if feasible (e.g. `$`, `\b`, i-or `(?<=\.)`).

Reply:
Good catch.

Reply:
re: anchors - I am not going to add them, I purposefully left them off